components/org.wso2.carbon.identity.conditional.auth.functions.http/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/http/SetCookieFunctionImpl.java

```diff
@@ -68,7 +71,9 @@ public void setCookie(JsServletResponse response, String name, Object... params)
             if (sign) {
                 try {
                     String tenantDomain = PrivilegedCarbonContext.getThreadLocalCarbonContext().getTenantDomain();
-                    signature = Base64.encode(IdentityUtil.signWithTenantKey(value, tenantDomain));
+                    HTTPFunctionsServiceHolder.getInstance().getIdentityKeyStoreGenerator()
```

Comment:
Lets add a comment mentioning what we are doing here.

Reply:
addressed in https://github.com/wso2/carbon-identity-framework/pull/6189/commits/6ca0b3d01b63c8f3f8ee122988af923b204f342b